packages/zowe-explorer/package.json

```diff
@@ -2079,12 +2107,10 @@
     "eslint-plugin-zowe-explorer": "3.0.0-next-SNAPSHOT",
     "event-stream": "^4.0.1",
     "expect": "^24.8.0",
-    "geckodriver": "^1.19.1",
     "glob": "^7.1.6",
-    "jsdom": "^16.0.0",
+    "jest-silent-reporter": "^0.5.0",
```

Comment:
Curious if this package is being used, since I don't see it referenced in the Jest config?

Reply:
It's not used in the current working tree, but I figured other contributors might find it useful to filter out a lot of the extraneous info from Jest results. I used this to clean up a bunch of failing tests without having to read in between the error outputs and the Jest metadata.

I can remove this if desired

Reply:
I'm not opposed to having it since you found it useful 😋 Could we mention somewhere how to use it - maybe a note somewhere in dev docs, or an npm script that runs tests in silent mode?

Reply:
Sure 😋 I'll add a script and a note in the wiki 👍

Reply:
As discussed offline, there isn't a good place to mention this on the wiki - but I've added the script in the latest commit.